src/ProductHub.Api/ProductHub.Api.csproj

```diff
@@ -16,6 +16,11 @@
   <ItemGroup>
     <ProjectReference Include="..\ProductHub.Application\ProductHub.Application.csproj" />
     <ProjectReference Include="..\ProductHub.Infrastructure\ProductHub.Infrastructure.csproj" />
+    <ProjectReference Include="..\ProductHub.Application.Contracts\ProductHub.Application.Contracts.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
Why

Reply:
Both Contract and API are part of presentation layer and communicates with application layer. I have split them in two separate projects to make it cleaner.